Album/AppDelegate.m

```diff
@@ -24,6 +25,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     NSString *key = [dict objectForKey: @"ggl_map_api_key"];
```

Comment:
for Keys you should maybe store them on a plist and preferably do not show that file on you repo (gitignore that plist) otherwise anyone can see keys that should be specific to your project

Reply:
100% agree.

I just double checked, and as I remembered Airei the Keys.plist is in the .gitignore. And I cannot find the "Keys.plist" in the repo.

Here she is accessing the .plist using a dictionary.  So, in the repo we access the google map key, by using the dictionary string key @"ggl_map_api_key".  

![Screen Shot 2022-08-04 at 11 16 31 PM](https://user-images.githubusercontent.com/21207111/183013894-62091ad2-241c-4b7c-ace3-f4211a0269a2.png)